prow/github/client.go

```diff
@@ -386,6 +386,24 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetLabels(org, repo string) ([]string, error) {
```

Comment:
Why do you say 10? 

[Get Labels](https://developer.github.com/v3/issues/labels/#list-all-labels-for-this-repository) indicates that it returns all for the repo.  Is it due to pagination, in which case the [default](https://developer.github.com/v3/#pagination) is 30 right?